core/src/main/java/dk/alexandra/fresco/lib/math/integer/exp/ExponentiationOpenExponent.java

```diff
@@ -29,7 +36,8 @@ public ExponentiationOpenExponent(DRes<SInt> x, BigInteger e) {
   public DRes<SInt> buildComputation(ProtocolBuilderNumeric builder) {
     return builder.seq((seq) -> {
       DRes<SInt> accEven = base;
-      return new IterationState(exponent, accEven, null);
+      DRes<SInt> accOdd = seq.numeric().known(BigInteger.ONE);
+      return new IterationState(exponent, accEven, accOdd);
```

Comment:
This gives quite unnecessary multiplication of known values.

Reply:
I redid your solution to save the multiplications

Reply:
Yikes, thanks for catching that!